server/schedule/checker/merge_checker.go

```diff
@@ -137,7 +137,7 @@ func (m *MergeChecker) Check(region *core.RegionInfo) []*operator.Operator {
 	log.Debug("try to merge region",
 		logutil.ZapRedactStringer("from", core.RegionToHexMeta(region.GetMeta())),
 		logutil.ZapRedactStringer("to", core.RegionToHexMeta(target.GetMeta())))
-	ops, err := operator.CreateMergeRegionOperator("merge-region", m.cluster, region, target, operator.OpMerge)
+	ops, err := operator.CreateMergeRegionOperator("merge-region", m.cluster, region, target, 0)
```

Comment:
Can we remove the argument `kind`?

Reply:
No, we can't because the waiting operator will divide the operators into different buckets according to the priority. If we remove it or just use `OpPlaceholder`, the two operators will be put into two kinds of buckets which is not expected when we manually add a merge operator through API.

Reply:
How about add a new function `NewMergeRegionOperator(cluster,source,target)`?  Just call`CreateMergeRegionOperator("merge-region",cluster,source,target,0)` in the function?